src/main/kotlin/com/faforever/userservice/security/FafJwtAuthenticationConverter.kt

```diff
@@ -13,15 +13,20 @@ data class FafAuthority(val role: String) : GrantedAuthority {
     override fun getAuthority() = "ROLE_$role"
 }
 
+data class FafScope(val role: String) : GrantedAuthority {
+    override fun getAuthority() = "SCOPE_$role"
```

Comment:
You have a const for SCOPE already. Also maybe move the scope class there

Reply:
Still not used to being able to reasonably put multiple classes in a single file

Reply:
Refactored